lib/src/importer-registry.ts

```diff
@@ -11,10 +11,13 @@ import {FileImporter, Importer, Options} from './vendor/sass';
 import * as proto from './vendor/embedded_sass_pb';
 import {catchOr, thenOr, PromiseOr} from './utils';
 
+const entryPointPathKey = Symbol();
+
 export class NodePackageImporter {
-  entryPointPath?: string;
+  [entryPointPathKey]?: string;
+
   constructor(entryPointPath?: string) {
-    this.entryPointPath = entryPointPath;
+    this[entryPointPathKey] = entryPointPath;
   }
```

Comment:
Per the latest proposal, this should now throw an error eagerly if `entryPointPath` isn't passed and `require.main.filename` doesn't exist.

Reply:
Addressed in https://github.com/oddbird/embedded-host-node/commit/f05aa50b54c51c5811deb27ca7cdf17cf6057576